pkg/ipamd/ipamd.go

```diff
@@ -340,11 +344,14 @@ func prometheusRegister() {
 	}
 }
 
-// containsInsufficientCidrBlocksError returns whether exceeds ENI's IP address limit
-func containsInsufficientCidrBlocksError(err error) bool {
+// containsInsufficientIPAddressError returns whether exceeds ENI's IP address limit
+func containsInsufficientIPAddressError(err error) bool {
```

Comment:
nit: Since this will error for /32 or /28, can we rename the function? Maybe `containsInsufficientCIDRs` or anything else. Also the comment needs to be rephrased, "returns whether a CIDR cannot be carved in the subnet or subnet is running out of IP addresses"